site/en/install/source.md

```diff
@@ -55,6 +55,8 @@ the build tool used to compile TensorFlow.
 
 Add the location of the Bazel executable to your `PATH` environment variable.
 
+Note: Make sure your Bazel version matches the minimum/maximum version as specified by `_TF_MIN_BAZEL_VERSION` and `_TF_MAX_BAZEL_VERSION` in `tensorflow/configure.py`.
```

Comment:
Link tensorflow/configure.py to GitHub file

Reply:
Sure, done in 4492330